index.html

```diff
@@ -375,26 +341,142 @@
 
     <section>
       <h2>Core Data Model</h2>
-      <p>This section describes a data model for <a>entity
-      profiles</a> and <a>entity credentials</a>, the latter covering
+
+      <p>
+This section outlines core data model concepts used throughout this
+specification
+      </p>
+
+      <section>
+        <h3>Claims</h3>
+
+        <p>
+A <a>subject</a> is an <a>entity</a> about which <a>claims</a> may be made.
+A <a>claim</a> is statement made by an <a>entity</a> about a <a>subject</a>.
+<a>Claims</a> are expressed using
+<strong><em>subject</em></strong>-<dfn data-lt="property|properties">property</dfn>-<dfn>value</dfn>
+associations.
+        </p>
+
+      <figure>
+        <img style="margin: auto; display: block;"
+          width="50%" src="diagrams/claim.svg">
+        <figcaption style="text-align: center;">
+The basic structure of a claim.
+        </figcaption>
+      </figure>
+
+      <p>
+The data model for <a>claims</a> described above is powerful and can be used to
+express a large variety of statements. For example, whether or not someone is
+over the age of 21 may be expressed as follows:
+      </p>
+
+      <figure>
+        <img style="margin: auto; display: block;"
+          width="50%" src="diagrams/claim-example.svg">
+        <figcaption style="text-align: center;">
+An example of a basic claim that expresses that Pat is over the age of 21.
+        </figcaption>
+      </figure>
+
+      <p>
+These claims may be merged together to express a mathematical graph of
+information about a particular subject. The example below extends the
+data model above by adding claims that state that Pat knows Sam and that
+Sam is a student.
+      </p>
+
+      <figure>
+        <img style="margin: auto; display: block;"
+          width="75%" src="diagrams/claim-extended.svg">
+        <figcaption style="text-align: center;">
+Multiple claims may be combined to express a more complex mathematical graph.
+        </figcaption>
+      </figure>
+
+      <p>
+At this point, the concept of a <a>claim</a> has been introduced. To make the
+claim verifiable, more information must be added to the graph of information.
+      </p>
+
+    </section>
+    <section>
+      <h2>Credentials</h2>
+
+      <p>
+A <a>credential</a> is set of one or more <a>claims</a> about a <a>subject</a>.
+It typically includes am identifier to uniquely identify the
```

Comment:
Fixed in https://github.com/w3c/vc-data-model/pull/67/commits/e1f1b82dcbd0124c388f3f23cc663707059c144d